.github/workflows/publish.yml

```
@@ -180,7 +180,12 @@ jobs:
       uses: maierj/fastlane-action@v1.4.0
       with:
         lane: build
-        options: '{ "flavor": "icm_ph_chc" }'        
+        options: '{ "flavor": "icm_ph_chc" }'
+    - name: Assemble vhtapp_uganda
+        uses: maierj/fastlane-action@v1.4.0
+        with:
+          lane: build
+          options: '{ "flavor": "vhtapp_uganda" }'
```

```suggestion
      uses: maierj/fastlane-action@v1.4.0
      with:
        lane: build
        options: '{ "flavor": "vhtapp_uganda" }'
```

The indentation seems to be wrong, with 2 spaces more. The CI process didn't fail because the publishing pipeline is executed only when tagging, but may fail once a new release is created.